Correct word usage (#26566)
mdn/content: files/en-us/learn/common_questions/web_mechanics/pages_sites_servers_and_search_engines/index.md

{{QuicklinksWithSubPages("Learn/Common_questions")}}

> In this article, we describe various web-related concepts: webpages, websites, web servers, and search engines. These terms are often confused by newcomers to the web or are incorrectly used. Let's learn what they each mean!

Author: ```suggestion
In this article, we describe various web-related concepts: web pages, websites, web servers, and search engines. These terms are often confused by newcomers to the web or are incorrectly used. Let's learn what they each mean!
```

We'll cover these terms and technologies in more detail as we explore further, but these quick definitions will be a great start for you:

> - webpage

Author: ```suggestion
- web page
```

So, let's dig deeper into how those four terms are related and why they are sometimes confused with each other.

> ### Webpage

Author: ```suggestion
### Web page
```

> A **webpage** is a simple document displayable by a {{Glossary("browser")}}. Such documents are written in the {{Glossary("HTML")}} language (which we look into in more detail in [other articles](/en-US/docs/Web/HTML)). A web page can embed a variety of different types of resources such as:

Author: ```suggestion
A **web page** is a simple document displayable by a {{Glossary("browser")}}. Such documents are written in the {{Glossary("HTML")}} language (which we look into in more detail in [other articles](/en-US/docs/Web/HTML)). A web page can embed a variety of different types of resources such as:
```

![Example of a web page address in the browser address bar](web-page.jpg)

### Website

> A _website_ is a collection of linked webpages (plus their associated resources) that share a unique domain name. Each webpage of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.

Author: ```suggestion
A _website_ is a collection of linked web pages (plus their associated resources) that share a unique domain name. Each web page of a given website provides explicit links—most of the time in the form of clickable portions of text—that allow the user to move from one page of the website to another.
```

To access a website, type its domain name in your browser address bar, and the browser will display the website's main web page, or _homepage_ (casually referred as "the home"):

![Example of a web site domain name in the browser address bar](web-site.jpg)

> Note that it also possible to have a  a _single-page website_: a site that consists of a single web page which is dynamically updated with new content when needed.

Author: ```suggestion
Note that it also possible to have a _single-page website_: a site that consists of a single web page which is dynamically updated with new content when needed.
```

> The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._

Reviewer: I don't find this sentence useful - it feels like we're comparing things that are not confusing in order to make them confusing. Also the `webpage` at the end - is that intentionally not spaced?

```suggestion
Note that it also possible to have a  a _single-page website: a site that consists of a single web page which is dynamically updated with new content when needed.
```

> The ideas of a _webpage_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._

Author: ```suggestion
The ideas of a _web page_ and a _website_ are especially easy to confuse for a _website_ that contains only one _webpage._ Such a website is sometimes called a _single-page website._
```
